### PR TITLE
[Workers] streams revision

### PR DESCRIPTION
**Pull request by deadlypants1973**

### CODE REVIEW

File: products/workers/src/content/learning/using-streams.md [hunk 1]
```diff
@@ -7,22 +7,22 @@ pcx-content-type: concept
...
 - The visitor’s time-to-first-byte.
 - The buffering done in the Workers script.
 
 Minimizing buffering is especially important for processing or transforming response bodies larger than the Workers script memory limit. For these cases, streaming is the only implementation strategy.
 
-<Aside>
+<Aside type="note">
 
-__Note:__ By default, the Cloudflare Workers service streams. Only use these APIs for _modifying_ the response body while maintaining streaming behavior. If your Workers script only passes subrequest responses back to the client verbatim without reading their body text, then its body handling is already optimal and you don’t have to use these APIs.
+By default, the Cloudflare Workers service streams. Only use these APIs for modifying the response body while maintaining streaming behavior. If your Workers script only passes subrequest responses back to the client verbatim without reading their body text, then its body handling is already optimal and you do not have to use these APIs.
```
Comment (deadlypants1973) [inline products/workers/src/content/learning/using-streams.md]: @lukeed "the Cloudflare Workers service streams."

Comment (lukeed) [reply products/workers/src/content/learning/using-streams.md:19]: Cloudflare Workers is capable of streaming responses using the [Streams APIs](https://developer.mozilla.org/en-US/docs/Web/API/Streams_API)

Comment (lukeed) [inline products/workers/src/content/learning/using-streams.md]:
```suggestion
By default, Cloudflare Workers is capable of streaming responses using the [Streams APIs](https://developer.mozilla.org/en-US/docs/Web/API/Streams_API). Only use these APIs for modifying the response body while maintaining streaming behavior. If your Worker only forwards subrequest responses to the client verbatim without reading their body text, then its body handling is already optimal and you do not have to use these APIs.
```

File: products/workers/src/content/learning/using-streams.md [hunk 2]
```diff
@@ -7,22 +7,22 @@ pcx-content-type: concept
...
 - The visitor’s time-to-first-byte.
 - The buffering done in the Workers script.
 
 Minimizing buffering is especially important for processing or transforming response bodies larger than the Workers script memory limit. For these cases, streaming is the only implementation strategy.
 
-<Aside>
+<Aside type="note">
 
-__Note:__ By default, the Cloudflare Workers service streams. Only use these APIs for _modifying_ the response body while maintaining streaming behavior. If your Workers script only passes subrequest responses back to the client verbatim without reading their body text, then its body handling is already optimal and you don’t have to use these APIs.
+By default, Cloudflare Workers is capable of streaming responses using the [Streams APIs](https://developer.mozilla.org/en-US/docs/Web/API/Streams_API). Only use these APIs for modifying the response body while maintaining streaming behavior. If your Worker only forwards subrequest responses to the client verbatim without reading their body text, then its body handling is already optimal and you do not have to use these APIs.
```
Comment (pedrosousa) [inline products/workers/src/content/learning/using-streams.md]:
```suggestion
By default, Cloudflare Workers is capable of streaming responses using the [Streams APIs](https://developer.mozilla.org/en-US/docs/Web/API/Streams_API). To maintain the streaming behavior, you should only modify the response body using the methods in the Streams APIs. If your Worker only forwards subrequest responses to the client verbatim without reading their body text, then its body handling is already optimal and you do not have to use these APIs.
```
Nit: MDN calls it the Streams API (singular), but we call it Streams APIs (plural).

File: products/workers/src/content/learning/using-streams.md [hunk 3]
```diff
@@ -55,13 +55,13 @@ The runtime can continue running a function (`response.body.pipeTo(writable)`) a
...
 
 </Aside>
 
 --------------------------------
 
-## See also
+## Related resources
 
 - [Streams API Reference](/runtime-apis/streams)
 - [MDN’s Streams API documentation.](https://developer.mozilla.org/en-US/docs/Web/API/Streams_API)
```
Comment (pedrosousa) [inline products/workers/src/content/learning/using-streams.md]:
```suggestion
- [MDN’s Streams API documentation](https://developer.mozilla.org/en-US/docs/Web/API/Streams_API)
```

**Review by pedrosousa — APPROVED** [review]

**Pull request merged**